WcaOnRails/config/initializers/iframe_mail_interceptor.rb

```diff
@@ -0,0 +1,18 @@
+class IframeMailInterceptor
+  class << self
+    def delivering_email(message)
+      # Replace iframes with corresponding links.
+      message.body = message.body.decoded.gsub(/<iframe.*?src=['"](.*?)['"].*?<\/iframe>/) do
+        # Handle YT and GMaps differently by converting embedded URLs into normal ones.
+        url = $1.gsub(/(?<=www.youtube.com\/)embed\//, "watch?v=")
+                .gsub(/(?<=www.google.com\/maps\/)embed\/v1\/place\?key=.*?q=/, "search/")
```

Comment:
Clever use of the zero width lookahead assertion! Took me a minute to understand this =)

I was just Googling and learned the term "leaning toothpick syndrome", which we seem to be suffering from a little bit. Would `%r{...}` be cleaner here?